.stylelintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-  "extends": "stylelint-config-standard",
-  "plugins": [
-    "stylelint-scss"
-  ],
-  "ignoreFiles": ["**/*.ts", "**/*.tsx", "dist/*", "node_modules/*", "coverage/*"],
+  "extends": "stylelint-config-sass-guidelines",
   "rules": {
-    "at-rule-no-unknown": null,
-    "scss/at-rule-no-unknown": true
+    "color-named": null,
+    "selector-max-id": null,
+    "selector-no-qualifying-type": null,
+
+    "scss/at-import-partial-extension-blacklist": null,
+    "declaration-property-value-disallowed-list": null,
+    "selector-class-pattern": null
```

Comment:
I do think we might want to preserve our `"ignoreFiles"` field sans Typescript.

I'm not too familiar with Stylelint's rule extensions. Do our rules extend the `stylelint-config-standard` implicitly?

Reply:
yup! i created a `.stylelintignore` file since the `ignoreFiles` thing here wasn't actually working. 
- I believe stylelint is a lil finnicky about where `ignoreFiles` goes or something, so i didnt want to deal with it, and instead created the ignore file

Reply:
Sounds good then. Thanks for doing that.